src/App.jsx

```diff
@@ -3,26 +3,73 @@ import './App.scss';
 import './styles/general.scss';
 import { TodoList } from './components/TodoList';
 import { CurrentUser } from './components/CurrentUser';
+import { getTodos } from './api/api';
 
 class App extends React.Component {
   state = {
     todos: [],
     selectedUserId: 0,
+    query: '',
+    status: 'All',
```

Comment:
move this values to the state inside TodoList component, entire app shouldn't know about filtering values